podium/settings.py

```diff
@@ -58,10 +60,15 @@
 
 ROOT_URLCONF = 'podium.urls'
 
+CRISPY_TEMPLATE_PACK ='bootstrap3'
+
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [
+            os.path.join(BASE_DIR, 'templates'),
+            os.path.join(BASE_DIR, 'templates/talks'),
```

Comment:
You shouldn't need to specifically list subdirectories for templates.  Django will automatically search subdirectories based on the template name.